.github/workflows/int-test-workflow.yml

```diff
@@ -206,10 +205,10 @@ jobs:
         if: ${{ always() }}
         run: |
           make cluster-down
-      - name: Test Report
-        uses: dorny/test-reporter@v1
-        if: success() || failure()    # run this step even if previous step failed
-        with:
-          name: Integration Tests          # Name of the check run which will be created
-          path: inttest-*.xml         # Path to test results
-          reporter: jest-junit        # Format of test results
\ No newline at end of file
+      #- name: Test Report
```

Comment:
Nit: Same as above

Reply:
we need to still investigate why the test report creation fails here. i will keep it for sometime